meadServer/app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Re the date formatting code: http://stackabuse.com/how-to-format-dates-in-javascript/

In general something that many lines long in a for loop with the same conceptual purpose should be its own method